collect_app/src/main/res/layout/form_download_error.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_margin="@dimen/margin_small"
+    app:cardElevation="2dp"
+    xmlns:tool="http://schemas.android.com/tools">
+
+    <LinearLayout
```

Comment:
Again, this should probably be a `ConstraintLayout` I think? Is the idea that it's ok to use the other layouts when there's no nesting?

Reply:
I just copied this layout from https://material.io/components/cards/android#card

Reply:
Cool. I'm not overly bothered by it. This makes more sense as a `LinearLayout` to me anyway.